src/psbt/finalizer.rs

```diff
@@ -216,44 +211,20 @@ pub fn interpreter_check<C: secp256k1::Verification>(
             .as_ref()
             .unwrap_or(&empty_witness);
 
+        println!("spk : {}", spk);
+        println!("ssi : {}", script_sig);
```

Comment:
Maybe you forgot to remove debug print

Reply:
Oh, lol, I saw those but totally forgot they were mine